### ISSUE TITLE
Countdown Timer when Clearing.

### ISSUE THREAD

**Issue by Wissi** [axle2005/ClearMob#25]
Great Plugin, we love it.
One request, could it be possible to add a config option for a countdown timer (or 5 messages in 1 sec delay) to let players know when the clear is going to happen.
Like: 
```
Clearing items in 5 secs
Clearing items in 4 secs
...
Clearing items in 1 sec
```
Thanks.

**Comment by axle2005** [comment]
I am someone who is completely against chat spam. I personally feel that giving them a minute heads up is more than sufficient.

**Comment by Wissi** [comment]
Maybe an option to change the minute (60 secs) to another value?
Just one msg still but configurable to another value?

Still love your work. Thanks.

**Comment by Trnrr** [comment]
I agree with something like this. On a server with a lot of stuff going on it is easy to miss messages. It would be nice to have the message on a configured timer. Maybe 60 secs, 30 secs and 10 secs? Or let it be configured.

**Comment by Yorkforce** [comment]
A 3 second countdown was very useful on a previous plugin, a configurable timer for messages would be nice.
